lib/util/pssh.js

```diff
@@ -90,16 +90,22 @@ shaka.util.Pssh = class {
   }
 
   /**
-   * Creates a pssh blob from the given system ID and data.
+   * Creates a pssh blob from the given system ID, data, keyIds and version.
    *
    * @param {!Uint8Array} data
    * @param {!Uint8Array} systemId
+   * @param {!Set.<string>} keyIds
+   * @param {number} version
    * @return {!Uint8Array}
    */
-  static createPssh(data, systemId) {
+  static createPssh(data,
```

Comment:
I think all the parameters should go into one line. There is no need to have two lines.

Reply:
Sounds good. Will make the change.
Thought there was a line length limit.

Reply:
The line limit is 80 characters.  :-)